api/app/Modules/Accounts/Http/Controllers/LibraryController.php

```diff
@@ -27,11 +27,9 @@ public function tracks(Request $request): JsonResponse
         return $this->respondWithPaginator($tracks, new TrackTransformer());
     }
 
-    public function getTrackIds()
+    public function getTrackIds(): JsonResponse
     {
-        $tracks = $this->getUser()->savedTracks()->get(['saveable_id'])->map(function ($result) {
-            return $result->saveable_id;
-        })->unique()->values();
+        $tracks = $this->getUser()->savedTracks()->get(['saveable_id'])->unique()->values();
```

Comment:
We need to revert this change and address whatever linting error you were facing

Reply:
Change reverted